core/gui/src/app/dashboard/type/workflow-executions-entry.ts

```diff
@@ -31,3 +32,12 @@ export interface WorkflowExecutionsEntry {
   bookmarked: boolean;
   logLocation: string;
 }
+
+export const EXECUTION_STATUS_CODE: Record<number, string> = {
```

Comment:
I wonder if we can use `ExecutionState` to replace those raw strings.

Reply:
changed